base/kcconf.py

```diff
@@ -10,7 +10,7 @@ def configkopano(configs):
     """ Changes configuration files according to configs typically returned from parseenvironmentvariables(..)"""
     for filename, config in configs.items():
         if not os.path.exists(filename):
-            return
+            continue
```

Comment:
I think this was added to prevent the script from erroring out of the file was not writeable

Reply:
not that much into Python myself, but just looked at the file history and the return was there already for the last two years, the change that I remembered for better handling of write protected files was more frequent though. So I am fine with this change.